internal/accountstable/accountstable.go

```diff
@@ -1,12 +1,18 @@
 package accountstable
 
 import (
+	"bytes"
 	"database/sql"
 	"encoding/hex"
 	"errors"
 	"fmt"
 
-	"github.com/SIGBlockchain/project_aurum/internal/producer/src/accountinfo"
+	"github.com/SIGBlockchain/project_aurum/internal/accountinfo"
+	"github.com/SIGBlockchain/project_aurum/internal/block"
+	// "github.com/SIGBlockchain/project_aurum/internal/constants"
```

Comment:
- [ ] Remove comment

Reply:
my bad, got it now

Reply:
I found some sql statements that are new that need to use the constants. Currently changing that